andela/activo-api
# #158355526 Add save assets endpoint

#### What does this PR do?
- Enables creation of asset
#### Description of Task to be completed?
- Create an `asset` view and add class for carrying out CRUD operations on asset entity
- add an endpoint that handles creation of asset
- write tests to cover the created functionality
#### How should this be manually tested?
- Create a branch `ft-save-asset-158355526` and pull from its remote counterpart
- Activate the python environment by running `venv/bin/activate`
- Run `pip install -r requirements.txt` to install all dependencies
- In your terminal run `flask run` to start the application.
- Using Postman, make a `post` request to the URL `http://127.0.0.1:5000/api/v1/assets`. Add a dictionary data with fields: `serial`, `tag` and `asset_category_id`
- Ensure to provide authorisation token. You should receive a message that the asset is created. 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Developer should be able to save an asset-158355526
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/24465059/41938244-771a8776-798a-11e8-9028-5d4739ccadb6.png)
![image](https://user-images.githubusercontent.com/24465059/41938256-7fceac76-798a-11e8-8ec9-711024a11ce3.png)

![image](https://user-images.githubusercontent.com/24465059/41938279-89fd34a6-798a-11e8-90eb-a17475644bb6.png)

#### Questions:
N/A
### Postman Link
https://www.getpostman.com/collections/4eae42b17b235eb0b982
